_posts/en/newsletters/2020-07-22-newsletter.md

```diff
@@ -0,0 +1,195 @@
+  command and new `keysend` RPC described in this newsletter's *notable
+  code changes* section, among several other notable changes and
+  multiple bug fixes.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19109][] introduces further improvements to
+  [transaction-origin privacy][transaction origin wiki]. Building on
+  [Bitcoin Core #18861][bitcoin 18861 newsletter], this patch adds a
+  per-peer rolling bloom filter to track which transactions were recently
+  announced to the peer. When a peer requests a transaction, this upgrade
+  checks the filter before fulfilling the request and relaying the
+  transaction to prevent spy nodes from learning about the exact contents
+  of our mempool or the precise timing of when a transaction was received.
+
+- [Bitcoin Core #16525][] updates several RPCs to return transaction
+  version numbers as unsigned integers rather than signed integers.
+  Early versions of Bitcoin interpreted version numbers as signed
+  integers (and
+  [some][35e79ee733fad376e76d16d1f10088273c2f4c2eaba1374a837378a88e530005]
+  transactions in the chain did actually set negative version numbers),
+  but the introduction of [BIP68][] nSequence enforcement for
+  transactions with version 2 or higher specified that it would match
+  against transaction nVersion fields cast to unsigned integers.  It's
+  expected that any other future soft forks that use transaction
+  versions will specify the same rule, and so it makes sense for RPCs to
+  always return unsigned numbers for transaction versions.
+
+- [C-Lightning #3809][] adds support to C-Lightning for effective
+  sending of [multipath payments][topic multipath payments]---payments
+  which are split into several parts, with each part routed using a
+  different path.  In brief, the algorithm C-Lightning uses is to split
+  a payment into approximately 0.0001 BTC parts (each party having its
+  amount randomly fuzzed by plus or minus 10%).  If any sent part fails,
+  that part is split into two parts (roughly in half, plus or minus 10%)
+  and the two parts are resent.  The PR additionally adds a
+  `disable-mpp` configuration option that will prevent sending any
+  multipath payments; a parameter of the same name is also added to the
+  `pay` command to disable sending a multipath payment for that
+  particular attempt.
+
+- [C-Lightning #3825][] changes the [recently merged][news105 cl3775]
+  (but never released) `reserveinputs` RPC so that it no longer
+  generates a new [PSBT][topic psbt].  Instead, a new `fundpsbt` RPC is
+  introduced to create a new PSBT.  Now an existing PSBT may be passed
```

Comment:
Consider s/Now an existing PSBT may be passed/An existing PSBT may now be passed/